libcloud/test/compute/test_ecs.py

```diff
@@ -528,15 +536,27 @@ def test_ex_list_security_groups(self):
         self.assertEqual('2015-06-26T08:35:30Z', sg.creation_time)
 
     def test_ex_join_security_group(self):
-        ex_security_group_id_value = 'sg-28ou0f3xa'
-        result = self.driver.ex_join_security_group(self.fake_node,
-                                                    group_id=ex_security_group_id_value)
+        result = self.driver.ex_join_security_group(
+            self.fake_node, group_id=self.fake_security_group_id)
         self.assertTrue(result)
 
     def test_ex_leave_security_group(self):
-        ex_security_group_id_value = 'sg-28ou0f3xa'
-        result = self.driver.ex_leave_security_group(self.fake_node,
-                                                     group_id=ex_security_group_id_value)
+        result = self.driver.ex_leave_security_group(
+            self.fake_node, group_id=self.fake_security_group_id)
+        self.assertTrue(result)
+
+    def test_ex_delete_security_group_by_id(self):
+        result = self.driver.ex_delete_security_group_by_id(
+            group_id=self.fake_security_group_id)
+        self.assertTrue(result)
+
+    def test_ex_modify_security_group_by_id(self):
+        self.sg_name = 'name'
+        self.sg_description = 'description'
```

Comment:
I think I would have preferred to keep the hardcoded values 'name' and 'description' because that's less state to keep track of if at some point we want to parallelize the tests. But that's minor and I will stop pestering you :)

Reply:
Thank you for your patient guidance. 
Do you mean the following code ?
```python
    def test_ex_modify_security_group_by_id(self):
        sg_name = 'name'
        sg_description = 'description'
        result = self.driver.ex_modify_security_group_by_id(
             group_id=self.fake_security_group_id,
             name=sg_name,
             description=sg_description)
        self.assertTrue(result)
```

Meanwhile check is
```python
    def _ModifySecurityGroupAttribute(self, method, url, body, headers):
        params = {'RegionId': self.test.region,
                  'SecurityGroupId': self.test.fake_security_group_id,
                   'SecurityGroupName': 'name',
                   'Description': 'description'}
```

Reply:
(Regarding your function before the edit, yes, I meant just use 'description' and 'name' hardcoded without any variable, in both functions.)